utils/contributors.js

```diff
@@ -59,6 +59,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'bsp2569',
+        name: "Bhabani Shankar Panigrahi",
+        image: "Passport size photo.jpeg",
```

Comment:
No image file attached.
Remove the image path if you don't wish to add any image.